### PR TITLE
[AWS] Update default EMR and method for passing in spark env vars

### PR DESCRIPTION
**Pull request by psmadbec**
It was simpler to merely create two tracks which added the hadoop (script) and spark env vars separately. I also took the opportunity to update the default EMR image. The release should smooth out whatever issues that might cause as it fails fast and is pretty easy to solve.

### CODE REVIEW

File: src/main/scala/org/broadinstitute/dig/aws/Emr.scala [hunk 1]
```diff
@@ -48,12 +48,17 @@ object Emr extends LazyLogging {
...
-          case _            => configurations :+= new Configuration(export).export(env)
-        }
+      // add environment variables as export classification for hadoop (for Scripts)
+      configurations.find(_.classification == "hadoop-env") match {
+        case Some(config) => config.export(env)
+        case _            => configurations :+= new Configuration("hadoop-env").export(env)
+      }
+
+      // add environment variables as properties for spark (for PySpark)
+      val modifiedEnv = env.map { case (key, value) => "spark.yarn.appMasterEnv." + key -> value }
```
Comment (sagehen03) [inline src/main/scala/org/broadinstitute/dig/aws/Emr.scala:58]: I don't see `modifiedEnv` used anywhere.  Is there a side effect that's happening that I'm missing?

**Review by sagehen03 — APPROVED** [review]
One question about modifiedEnv var, but otherwise seems straightforward.

File: src/main/scala/org/broadinstitute/dig/aws/Emr.scala (hunk 1 above)
Comment (psmadbec) [reply src/main/scala/org/broadinstitute/dig/aws/Emr.scala:58]: `case Some(config) => config.addProperties(modifiedEnv)`

It is the two lines below, in the match-case.

Comment (sagehen03) [reply src/main/scala/org/broadinstitute/dig/aws/Emr.scala:58]: Sorry, not sure how I missed that!

**Pull request merged**